engine/runtime-instrument-common/src/main/scala/org/enso/interpreter/instrument/job/ProgramExecutionSupport.scala

```diff
@@ -409,13 +410,26 @@ object ProgramExecutionSupport {
           val schema = value.getValue match {
             case function: Function =>
               val functionInfo = FunctionPointer.fromFunction(function)
-              toMethodPointer(functionInfo).map { methodPointer =>
-                Api.FunctionSchema(
-                  methodPointer,
-                  FunctionPointer.collectNotAppliedArguments(function).toVector
+              val notAppliedArguments = FunctionPointer
+                .collectNotAppliedArguments(function)
+                .toVector
+              toMethodPointer(functionInfo).map(methodPointer =>
+                Api.FunctionSchema(methodPointer, notAppliedArguments)
+              )
+            case atomConstructor: AtomConstructor =>
+              val functionInfo = new FunctionPointer(
```

Comment:
Create `FunctionPointer.fromConstructor(...)` method and test it with https://github.com/enso-org/enso/issues/8663#issuecomment-1885125148 test.